qtpy/QtGui.py

```diff
@@ -33,7 +33,10 @@
                                  qFuzzyCompare)
     except ImportError:
         pass
-    from PyQt4.Qt import QKeySequence, QTextCursor
+    try:
```

Comment:
Could you add some comments here on why this is needed like:

```
# PyQt 4.x.x requires .....
```

Reply:
Sure, I will

Reply:
Thanks :-)